src/docs/sphinx/advancedExamples/validationStudies/wellboreProblems/casedContactThermoElasticWellbore/Example.rst

```diff
@@ -0,0 +1,156 @@
+.. _AdvancedExampleCasedContactThermoElasticWellbore:
+
+
+######################################################################
+Cased ThermoElastic Wellbore Problem with Imperfect Contact Interfaces
+######################################################################
+
+------------------------------------------------------------------
+Problem description
+------------------------------------------------------------------
+
+This example uses the coupled THM solver to handle a cased wellbore problem subjected to a cold temperature condition at the casing surface because of cold CO2 injection. The completed wellbore is composed of a steel casing, a cement sheath and rock formation. Isotropic linear thermo-elastic behavior is assumed for all the three materials. Fluid flow is not considered, i.e. heat tranfer between the casing, cement layer and rock formation is purely diffusion. Heat transfer by the convexion phenominon is not negligible. The capability of GEOS is not limited to this case, however we chose to simulate this simplified case because of the exisiting of the semi-analytical results. The separation is allowed for the casing-cement and cement-rock contact interfaces.
```

Comment:
```suggestion
This example uses the coupled THM solver to handle a cased wellbore problem subjected to temperature reduction at the casing surface due to cold CO2 injection. The completed wellbore is composed of a steel casing, a cement sheath and rock formation. Isotropic linear thermo-elastic behavior is assumed for all the three materials. Fluid flow is not considered, i.e. heat transfer between the casing, cement layer and rock formation is pure conduction. Heat transfer by the convection phenomenon is not considered. The capability of GEOS is not limited to this case, however we chose to simulate this simplified case because of the existing semi-analytical solutions. The separation is allowed for the casing-cement and cement-rock contact interfaces.
```

Reply:
done